[2025-02-11T21:51:58.069+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-11T21:51:58.103+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data manual__2025-02-11T21:51:34.854098+00:00 [queued]>
[2025-02-11T21:51:58.123+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data manual__2025-02-11T21:51:34.854098+00:00 [queued]>
[2025-02-11T21:51:58.125+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-11T21:51:58.154+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): transform_load_weather_data> on 2025-02-11 21:51:34.854098+00:00
[2025-02-11T21:51:58.169+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=227) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-11T21:51:58.174+0000] {standard_task_runner.py:72} INFO - Started process 229 to run task
[2025-02-11T21:51:58.173+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'transform_load_weather_data', 'manual__2025-02-11T21:51:34.854098+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpfczag08a']
[2025-02-11T21:51:58.180+0000] {standard_task_runner.py:105} INFO - Job 36: Subtask transform_load_weather_data
[2025-02-11T21:51:58.272+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_dag.transform_load_weather_data manual__2025-02-11T21:51:34.854098+00:00 [running]> on host dbef5e540f20
[2025-02-11T21:51:58.451+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='aida' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='transform_load_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-11T21:51:34.854098+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-11T21:51:34.854098+00:00'
[2025-02-11T21:51:58.455+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-11T21:51:58.641+0000] {logging_mixin.py:190} INFO -        City Description  ...  Sunrise (Local Time)  Sunset (Local Time)
0  Portland   clear sky  ...   2025-02-11 07:18:10  2025-02-11 17:31:53

[1 rows x 12 columns]
[2025-02-11T21:51:58.658+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-11T21:51:58.683+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-11T21:51:58.685+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=transform_load_weather_data, run_id=manual__2025-02-11T21:51:34.854098+00:00, execution_date=20250211T215134, start_date=20250211T215158, end_date=20250211T215158
[2025-02-11T21:51:58.763+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-11T21:51:58.811+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-11T21:51:58.821+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
